CHANGELOG.md

```diff
@@ -8,13 +8,19 @@ All notable changes to this project will be documented in this file.
 
 - Support enabling decision logs ([#555]).
 
+### Changed
+
+- Bump `product-config` from `0.6.0` to `0.7.0` ([#595]).
+- Bump `stackable-operator` from `0.69.3` to `0.70.0` ([#595]).
```

Comment:
Other dependencies were updated, so I'd suggest collapsing into a single line, like:

```suggestion
- Bump `stackable-operator` to 0.70.0, `product-config` to 0.7.0, and other dependencies  ([#595]).
```

I also don't mind it being across multiple lines, but the other dependency updates should be mentioned, like:

```suggestion
- Bump `product-config` from `0.6.0` to `0.7.0` ([#595]).
- Bump `stackable-operator` from `0.69.3` to `0.70.0` ([#595]).
- Bump other dependencies ([#595]).
```